Gemfile

```diff
@@ -53,6 +53,9 @@ gem "clearance"
 # Pundit user authorization
 gem "pundit", "~> 2.5"
 
+# Down image download
+gem "down", "~> 5.0"
```

Comment:
Any chance you remember why you chose to use this gem? I've never used it before. I would think you have everything you need for retrieving an image without adding an extra dependency.